plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1025,7 +1037,13 @@ func (c *PodAffinityChecker) getMatchingAntiAffinityTerms(pod *v1.Pod, allPods [
 				return nil, err
 			}
 			for _, term := range getPodAntiAffinityTerms(affinity.PodAntiAffinity) {
-				match, err := priorityutil.PodMatchesTermsNamespaceAndSelector(pod, existingPod, &term)
+
+				namespaces := priorityutil.GetNamespacesFromPodAffinityTerm(existingPod, term)
```

Comment:
Please be consistent - either always compute namespaces first or always compute selector first.